extensions/openshift-client/deployment/src/main/java/io/quarkus/openshift/client/deployment/OpenShiftClientProcessor.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.openshift.client.deployment;
+
+import javax.inject.Inject;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.it.openshift.client.OpenShiftClientProducer;
+
+public class OpenShiftClientProcessor {
+
+    private static final Logger log = Logger.getLogger(OpenShiftClientProcessor.class.getName());
+
+    @Inject
+    BuildProducer<FeatureBuildItem> featureProducer;
+
+    @BuildStep
+    public void process(
+            BuildProducer<ExtensionSslNativeSupportBuildItem> sslNativeSupport,
+            BuildProducer<AdditionalBeanBuildItem> additionalBeanBuildItemBuildItem) {
+
+        featureProducer.produce(new FeatureBuildItem(Feature.OPENSHIFT_CLIENT));
+
+        // Enable SSL support by default
+        sslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(Feature.OPENSHIFT_CLIENT));
```

Comment:
The kubernetes client extension already does that

Reply:
it does this:
```
featureProducer.produce(new FeatureBuildItem(Feature.KUBERNETES_CLIENT));
sslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(Feature.KUBERNETES_CLIENT));
```

not this:
```
featureProducer.produce(new FeatureBuildItem(Feature.OPENSHIFT_CLIENT));
sslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(Feature.OPENSHIFT_CLIENT));
```

don't we want the feature to be displayed?
and for ssl support I agree.
it was not clear to me whether or not things needed to be repeated in the context of the new extension (I am not entirely comfortable with processors).

Reply:
Yes, I was referring to the SSL support. Thanks!